docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -75,12 +58,26 @@ The runner can optionally accept the following keys.
   A URL for a remote survey JSON. This claim is used to tell Survey Runner to load the schema JSON from a remote location
 ``case_ref``
   The case reference identified by the above UUID (e.g. "1000000000000001")
+``case_type``
+  The type of case
+``account_service_url``
+  The url of the account service used to launch the survey
 ``account_service_log_out_url``
   The logout url of the account service used to launch the survey.  Not required for services that don't have a log in function (i.e., respondent home)
+``channel``
+  The channel (client) from which the questionnaire was launched
+
+Schema Metadata
+======================
+
+In addition to the above required fields, some surveys require other data be passed. These can simply have their keys added as a claim in the main JWT body. e.g.
+
+``display_address``
+  The case's address to be displayed
```

Comment:
Currently the `user_id` and `period_id` are also defined in schema metadata. If they are required regardless of schema, then the runner implementation should probably change, if not then they should be documented here instead.

Reply:
They were required for the Census and it looks like the [converter build](https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/app/submitter/converter.py#L116-L121) functions still expects these to be present. We can discuss the correct approach to achieving that separately to this PR